tinyagentos/chat/message_store.py

```
@@ -92,6 +92,28 @@ async def init(self) -> None:
             await self._db.commit()
         except Exception:
             pass
+        # E1 collab: remote message dedup columns
+        try:
+            await self._db.execute("ALTER TABLE chat_messages ADD COLUMN remote_msg_id TEXT")
+            await self._db.commit()
+        except Exception:
+            pass
```

Ruff (0.15.21)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)